Clean up `Code of conduct for modding and mapping` (#6054)
ppy/osu-wiki: wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/en.md

## Behaviour and conduct

- The goal of modding is to help a mapper improve their work so that it can be pushed towards [ranked](/wiki/Beatmap/Category#ranked) status if the mapper chooses to do so. If you're not interested in helping people, steer clear of the discussion to avoid *hurting them*.
- When modding, consider who the post is directed at, what you aim to achieve by posting and what response you expect to receive. This will ensure your best intentions are conveyed and your post is easy to understand.

>   - Don't shy away from new mappers and modders - these people are the ones who can benefit the most from your input! Be patient, understanding, and provide as much explanation as possible so that they can learn and improve more quickly

Reviewer: also em dashes

```suggestion
  - Don't shy away from new mappers and modders—these people are the ones who can benefit the most from your input! Be patient, understanding, and provide as much explanation as possible so that they can learn and improve more quickly
```

  - If you're new yourself, don't be afraid to say so! This will allow more experienced community members to point you in the right direction. Everyone was new and in your position once!

> - Treat others as you wish to be treated. People are more receptive to kind, helpful suggestions than hostile ones. Try to understand others' point-of-view rather than attack it - nobody likes to feel worthless, incompetent or stupid. If you are a masochist, assume that all others are not.

Reviewer: ```suggestion
- Treat others as you wish to be treated. People are more receptive to kind, helpful suggestions than hostile ones. Try to understand others' point-of-view rather than attack it—nobody likes to feel worthless, incompetent or stupid. If you are a masochist, assume that all others are not.
```

> - If you wish to comment on or discuss a beatmap, go to that beatmap's [discussion page](/wiki/Beatmap_Discussion) - not social media outlets, or the comment section, as it is unlikely your concerns will be heard by anyone with the power to address them appropriately.

Reviewer: ```suggestion
- If you wish to comment on or discuss a beatmap, go to that beatmap's [discussion page](/wiki/Beatmap_Discussion)—not social media outlets, or the comment section, as it is unlikely your concerns will be heard by anyone with the power to address them appropriately.
```

- Personal attacks, hateful comments, misplaced joking, derailing the thread and general drama-initiation are not acceptable in any form and **will not be tolerated**.

## Making a mod post

> So you downloaded a map and you've thought of some things you aren't happy with - now's the time to put your thoughts to words and help improve the map. But how can you get the mapper to see your point of view, test your suggestions and give you feedback? After all, no one likes feeling that their time has been wasted.

Reviewer: ```suggestion
So you downloaded a map and you've thought of some things you aren't happy with—now's the time to put your thoughts to words and help improve the map. But how can you get the mapper to see your point of view, test your suggestions and give you feedback? After all, no one likes feeling that their time has been wasted.
```

# Code of conduct for modding and mapping

> The **Code of conduct for modding and mapping** is a set of rules and guidelines that apply to the entirety of the osu! [modding](/wiki/Modding) and [mapping](/wiki/Beatmapping) ecosystem and an extension to the [osu! community rules](/wiki/Rules) and [general ranking criteria](/wiki/Ranking_Criteria). In order to ensure that mapping and modding discussions in threads take place in a constructive, positive and productive environment, a code of conduct is crucial to get everyone on the same page. When participating in the modding and mapping ecosystem, it is mandatory to follow this code of conduct. Misconduct that violates these rules might lead to [penalties issued to your account](/wiki/Silence).

Reviewer 2: ```suggestion
The **code of conduct for modding and mapping** is a set of rules and guidelines that apply to the entirety of the osu! [modding](/wiki/Modding) and [mapping](/wiki/Beatmapping) ecosystem and an extension to the [osu! community rules](/wiki/Rules) and [general ranking criteria](/wiki/Ranking_Criteria). In order to ensure that mapping and modding discussions in threads take place in a constructive, positive and productive environment, a code of conduct is crucial to get everyone on the same page. When participating in the modding and mapping ecosystem, it is mandatory to follow this code of conduct. Misconduct that violates these rules might lead to [penalties issued to your account](/wiki/Silence).
```

## Behaviour and conduct

> - The goal of modding is to help a mapper improve their work so that it can be pushed towards [ranked](/wiki/Beatmap/Category#ranked) status if the mapper chooses to do so. If you're not interested in helping people, steer clear of the discussion to avoid *hurting them*.

Reviewer 2: ```suggestion
- The goal of modding is to help a mapper improve their work so that it can be pushed towards the [ranked](/wiki/Beatmap/Category#ranked) status if the mapper chooses to do so. If you're not interested in helping people, steer clear of the discussion to avoid *hurting them*.
```

There are any number of ways to present your mod post, so feel free to experiment, but try to keep the visuals simple and formatted in a way that it is easy for the mapper and others to work with.

## Responding to a mod

> Someone just posted some suggestions on your map, great! - but it doesn't end there. For the record, it's necessary to give a proper response for two major reasons. Firstly, so that future mods can be tailored more specifically to your concept or style, resulting in fewer unhelpful suggestions. Secondly, just as a mod post *helps you improve your mapping ability*, providing a detailed mod response helps the modder improve their modding ability, meaning they gain experience and can provide more helpful and detailed mods in the future. You never know who you may need to ask for a mod next time around!

Reviewer: hmm, something tells me that dash isn't correct here..

```suggestion
Someone just posted some suggestions on your map, great! However, it doesn't simply end there. For the record, it's necessary to give a proper response for two major reasons. Firstly, so that future mods can be tailored more specifically to your concept or style, resulting in fewer unhelpful suggestions. Secondly, just as a mod post *helps you improve your mapping ability*, providing a detailed mod response helps the modder improve their modding ability, meaning they gain experience and can provide more helpful and detailed mods in the future. You never know who you may need to ask for a mod next time around!
```

Author: I'll take your word for English punctuation

> - Treat all suggestions as valid and equally important until you have thoroughly inspected and tested them - even minor stylistic changes or alternative viewpoints can help improve the quality of a beatmap

Reviewer: ```suggestion
- Treat all suggestions as valid and equally important until you have thoroughly inspected and tested them—even minor stylistic changes or alternative viewpoints can help improve the quality of a beatmap
```

- Treat all suggestions as valid and equally important until you have thoroughly inspected and tested them—even minor stylistic changes or alternative viewpoints can help improve the quality of a beatmap
- As with making a mod post, be sure to explain yourself as much as possible. State what you changed, and if you didn't accept the suggestion for some reason, explain why and what you did instead.
- When declining a suggestion, remain calm and professional, say what needs to be said in order to explain yourself and **leave it at that**. No matter how impolite, confusing or unhelpful the suggestion was, getting angry and responding with hostility, sarcasm, etc. will not help the situation.

> - [Beatmap nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) and [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT) members will review previous mods before placing any icons and may want to initiate or revisit discussions during their checks. Any assistance you can provide as to why you did or didn't do something takes very little effort, and could be a huge help to all parties later on.

Reviewer 2: ```suggestion
- [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (BN) and [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (NAT) members will review previous mods before placing any icons and may want to initiate or revisit discussions during their checks. Any assistance you can provide as to why you did or didn't do something takes very little effort, and could be a huge help to all parties later on.
```

## Qualified beatmaps

The [Qualified section](/wiki/Beatmap/Category#qualified) allows mapsets to receive any final suggestions for change or improvement before reaching Ranked status. When modding a Qualified map, everything you consider "not minor" should **always** be mentioned - minor issues are also worth mentioning however, especially if they affect gameplay. If a Qualified mapset accumulates a significant number of minor issues, or if there is one or more major / unrankable issues, the mapset should be disqualified so that fixes and improvements can be made before attempting requalification.
The [Qualified section](/wiki/Beatmap/Category#qualified) allows beatmaps to receive any final suggestions for change or improvement before reaching ranked status. When modding a qualified map, everything you consider "not minor" should **always** be mentioned—minor issues are also worth mentioning however, especially if they affect gameplay. If a qualified beatmap accumulates a significant number of minor issues, or if there is one or more major/unrankable issues, the beatmap should be disqualified so that fixes and improvements can be made before attempting requalification.

> If you're unsure on the nature of a potential problem, ask for other opinions or review previous mod posts to make sure the "issue" presented isn't something done intentionally by the mapper - if so, it will likely have been discussed already. If there has been no prior discussion, feel free to open one as if you were modding the map normally. If you feel the issues discovered should prevent the mapset from being ranked, report it in this thread with a link to your mod post and a brief outline of your concerns.

Reviewer 2: was this sentence removed intentionally? feels somehow important to have it there

Author: there are three sentences: 1) ask around if you are unsure, 2) open a discussion if the issue wasn't mentioned, and 3) report the issue if it's important

I've asked in #nat, and it turned out the thread was closed in favour of beatmap discussion, which means that 2) and 3) both lead to the beatmap discussion area and eventually cover the same flow. so, they could be replaced with a single sentence instead of the two saying the same thing

- If you can't explain something through logic, you will not be able to explain it to them through violence either. Rather than continue a discussion that isn't progressing, walk away and try again with the next modder/mapper.

> - When things look to be getting out of hand, or of the discussion is struggling to stay on topic, [call a moderator](/wiki/Reporting_Bad_Behaviour) - don't contribute to the problem by continuing to post.

Reviewer: ```suggestion
- When things look to be getting out of hand, or if the discussion is struggling to stay on topic, [call a moderator](/wiki/Reporting_Bad_Behaviour). Don't contribute to the problem by continuing to post.
```

## Qualified beatmaps

> The [Qualified section](/wiki/Beatmap/Category#qualified) allows beatmaps to receive any final suggestions for change or improvement before reaching ranked status. When modding a qualified map, everything you consider "not minor" should **always** be mentioned - minor issues are also worth mentioning however, especially if they affect gameplay. If a qualified beatmap accumulates a significant number of minor issues, or if there is one or more major/unrankable issues, the beatmap should be disqualified so that fixes and improvements can be made before attempting requalification.

Reviewer: ```suggestion
The [Qualified section](/wiki/Beatmap/Category#qualified) allows beatmaps to receive any final suggestions for change or improvement before reaching ranked status. When modding a qualified map, everything you consider "not minor" should **always** be mentioned—minor issues are also worth mentioning however, especially if they affect gameplay. If a qualified beatmap accumulates a significant number of minor issues, or if there is one or more major/unrankable issues, the beatmap should be disqualified so that fixes and improvements can be made before attempting requalification.
```

> The [Qualified section](/wiki/Beatmap/Category#qualified) allows beatmaps to receive any final suggestions for change or improvement before reaching ranked status. When modding a qualified map, everything you consider "not minor" should **always** be mentioned - minor issues are also worth mentioning however, especially if they affect gameplay. If a qualified beatmap accumulates a significant number of minor issues, or if there is one or more major / unrankable issues, the beatmap should be disqualified so that fixes and improvements can be made before attempting requalification.

Reviewer: ```suggestion
The [Qualified section](/wiki/Beatmap/Category#qualified) allows beatmaps to receive any final suggestions for change or improvement before reaching ranked status. When modding a qualified map, everything you consider "not minor" should **always** be mentioned - minor issues are also worth mentioning however, especially if they affect gameplay. If a qualified beatmap accumulates a significant number of minor issues, or if there is one or more major/unrankable issues, the beatmap should be disqualified so that fixes and improvements can be made before attempting requalification.
```

> If you're unsure on the nature of a potential problem, ask for other opinions or review previous mod posts to make sure the "issue" presented isn't something done intentionally by the mapper - if so, it will likely have been discussed already. If there has been no prior discussion, feel free to open one as if you were modding the map normally, and post a brief outline of your concerns.

Reviewer: ```suggestion
If you're unsure on the nature of a potential problem, ask for other opinions or review previous mod posts to make sure the "issue" presented isn't something done intentionally by the mapper—if so, it will likely have been discussed already. If there has been no prior discussion, feel free to open one as if you were modding the map normally, and post a brief outline of your concerns.
```

> **Note that it is the community's responsibility alone to report issues with qualified beatmaps**. It is not guaranteed that BNs or NAT members will actively check and disqualify beatmaps without being prompted to do so.

Reviewer: just for the sake of removing the plural on the acronym

```suggestion
**Note that it is the community's responsibility alone to report issues with qualified beatmaps**. It is not guaranteed that any BN or NAT member will actively check and disqualify beatmaps without being prompted to do so.
```

Author: good call
